### ISSUE TITLE
EIV / Adjusted Price

### ISSUE THREAD

**Issue by SubliminallyFamous** [nuadi/googlecrestscript#42]
Hi Nuadi, I'm having some issues with the Adjusted Price / Estimated Item Value.
I understand the EIV of an item is the Adjusted Price of the minerals required to build it (at an ME of 0%). 
This is more or less spot on for sub capital ships (to a hundred ISK or so due to the small delay from what the script pulls and what you see live in-game), however when calculating for a capital hull the value pulled is far lower than what it displays in game.
The EIV for an Apostle showed at 1,234,315,785 ISK yesterday in-game, but the value pulled from the script was 742,446,815.71 ISK for the HULL and 891,329,055.51 ISK for the minerals required to build the hull.
Am I missing a step in the calculations for capital hulls? The same formula applied to Sub Capital Hulls seems accurate.
The formula I'm using for calculating job costs is:
EIV*INDEX - ROLE + TAX * QTY
Or:
=((((1234315785*3.5741129918403%)-((1234315785*3.5741129918403%)*5%))+((1234315785*3.5741129918403%)*0.2%)*1))

Cheers
Sub

**Comment by nuadi** [comment]
>and 891,329,055.51 ISK for the minerals

I don't believe a capital uses minerals, correct? So, did you mean you did in fact add up the cost of minerals, or did you pull the adjusted price for the components?

**Comment by SubliminallyFamous** [comment]
Hi Nuadi, I've also added up the adjusted price for the components required to build the hull and this comes to 891,329,055.51 - exactly the same as what is pulled via the script for the hull, but of course this doesn't match what the EIV displays in-game.

Edit: I may have noticed a mistake in some caluclations, unfortunately ESI is down at the moment but will update here again once I've re-run the maths after ESI is back up. Thanks for getting back to me so quickly.

**Comment by SubliminallyFamous** [comment]
Hi Naudi, I found where I went wrong:
You essentially have to calculate an adjusted/EIV price for components twice.
When building the components you calculate the adjusted price of the minerals required to build them and when building the hull you calculate the adjusted price of the components required to build the hull.
A refresh and recalculation and my values came out at the 1,234,315,785.28 ISK value - bang on target.
Thanks again
Sub
